src/_common/Catalog/IndicatorUtility.cs

```diff
@@ -0,0 +1,252 @@
+using System.Globalization;
+using System.Text.Json;
+
+namespace Skender.Stock.Indicators;
+
+/// <summary>
+/// Utility class for executing indicators with minimal configuration requirements.
+/// Provides simple methods to execute indicators using only ID and Style without requiring
+/// prior knowledge of result types or implementation syntax.
+/// </summary>
+public static class IndicatorUtility
+{
+    /// <summary>
+    /// Executes an indicator using only its ID and style.
+    /// </summary>
+    /// <param name="id">The indicator ID (e.g., "EMA", "RSI", "MACD").</param>
+    /// <param name="style">The indicator style (Series, Stream, or Buffer).</param>
+    /// <param name="quotes">The quotes to process.</param>
+    /// <param name="parameters">Optional parameter overrides.</param>
+    /// <returns>The indicator results as a list of objects.</returns>
+    /// <exception cref="ArgumentNullException">Thrown when quotes is null.</exception>
+    /// <exception cref="ArgumentException">Thrown when id is null or empty.</exception>
+    /// <exception cref="InvalidOperationException">Thrown when the indicator cannot be found or executed.</exception>
+    public static IReadOnlyList<object> ExecuteById(
+        string id,
+        Style style,
+        IEnumerable<IQuote> quotes,
```

Comment:
Use the same extension pattern we use for series indicators `this IEnumerable`, so this can be called as `quotes.ExecuteById(..)`